base/pop/getpoploglib/lib/getpoplog_run_subcommand.p

```diff
@@ -0,0 +1,62 @@
+compile_mode :pop11 +strict;
+
+section;
+
+uses getpoplog;
+
+;;; Given command line options of the form
```

Comment:
nit: seems a bit odd that short args with a single dash would end up in the second list.

Reply:
Oh so sharp-eyed! Yes, this is a pretty crude first attempt at processing command line options. It needs more work and that is why it isn't exposed as a library as yet. Another infelicity in the same vein is that there is no way to turn off option processing.

I will raise a suitable issue.